mito-ai/mito_ai/completions/prompt_builders/agent_system_message.py

```diff
@@ -72,6 +74,7 @@ def create_agent_system_message_prompt(isChromeBrowser: bool) -> str:
         type: 'new'
         index: int
         code: str   
+        code_summary: str
```

Comment:
There's some cell_update examples in this large system prompt that we forgot to add `code_summary` to. Can you double check that we add it in all of the cell_updates.